scripts/rest_examples/http-client.env.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dev": {
+    "hostname": "localhost",
+    "port": "8000",
+    "prefix": "api/provisioning",
+    "identity": "eyJpZGVudGl0eSI6IHsidHlwZSI6ICJVc2VyIiwgImFjY291bnRfbnVtYmVyIjoiMTMiLCAiaW50ZXJuYWwiOnsib3JnX2lkIjoiMDAwMDEzIn19fQo="
+  }
```

Comment:
We can probably create multiple of these and name them "user account 13 org_id 000013" or something like that so ppl can switch easily.